tests/setup_test.go

```diff
@@ -1928,3 +1929,136 @@ func TestUserOrgs(t *testing.T) {
 		t.Fatalf("user orgs mismatch (-want +got):\n%s", diff)
 	}
 }
+
+func TestTaskTimeout(t *testing.T) {
```

Comment:
Can you add some more complex tests with the timing out task that has both some parents and some child dependencies?

Reply:
Hi @sgotti I added some tests. Can you do the review please?